aten/src/ATen/native/mps/operations/Pooling.mm

```diff
@@ -1,5 +1,6 @@
 //  Copyright © 2022 Apple Inc.
 #define TORCH_ASSERT_ONLY_METHOD_OPERATORS
+#include <ATen/Dispatch.h>
```

Comment:
Unneded?
```suggestion
```

Reply:
yeap, I added this when I was trying to figure why it wasn't building